server/block/nether_wart_block.go

```diff
@@ -0,0 +1,30 @@
+package block
+
+// NetherWartBlock is a decorative block found in crimson forests and crafted using Nether wart.
+type NetherWartBlock struct {
```

Comment:
Would name this `NetherWart` for consistency with other blocks and rename the file too.